### PR TITLE
Hotfix/gh583

### PR DESCRIPTION
**Pull request by FrankMittelbach**
# Internal housekeeping

## Status of pull request
- Ready to merge

## Checklist of required changes before merge will be approved
- [x] Test file(s) added
- [x] Version and date string updated in changed source files
- [x] Relevant `\changes` entries in source included
- [x] Relevant `changes.txt` updated
- [n/a] `ltnewsX.tex` updated

### CODE REVIEW

File: base/doc/latexchanges.tex [hunk 1]
```diff
@@ -124,6 +125,11 @@ \section{Changes introduced in 2021-06-01 patch~1}
 Fixes an issue with rollback when \pkg{latexrelease} is loaded by some
 other package (as done for \texttt{platex} (\ghissue{577}).
 
+First aid for \pkg{ulem} and \pkg{varwidth} got added to help them
+along until the get corrected. \pkg{ulem} has a problem with the
```
Comment (davidcarlisle) [inline base/doc/latexchanges.tex:129]: they

File: required/firstaid/changes.txt [hunk 2]
```diff
@@ -1,3 +1,9 @@
+2021-06-08  Frank Mittelbach  <Frank.Mittelbach@latex-project.org>
+
+	* latex2e-first-aid-for-external-files.dtx:
+	ulem doesn't handle \hspace gracefully not that it is calc aware.
```
Comment (davidcarlisle) [inline required/firstaid/changes.txt:4]: now   (not not)

File: required/firstaid/latex2e-first-aid-for-external-files.dtx [hunk 3]
```diff
@@ -546,6 +548,70 @@
...
+   \def\@hspace#1{\begingroup\setlength\skip@{#1}%
+                  \edef\x{\endgroup\hskip\the\skip@\relax}\x}%
+   }
+%    \end{macrocode}
+%
+%
+% \subsection{\cs{varwidth} first aid}
+%
+%    the \pkg{varwidth} package does a lot of low-level paragraph
+%    manipulation assuming tradition \TeX{} paragraphs. However with
```
Comment (davidcarlisle) [inline required/firstaid/latex2e-first-aid-for-external-files.dtx:571]: traditional

**Review by davidcarlisle — APPROVED** [review]
Code looks fine Just a couple of typo comments

File: required/firstaid/latex2e-first-aid-for-external-files.dtx [hunk 4]
```diff
@@ -546,6 +548,70 @@
...
+%    the paragraph hooks we end up with one extra glue 0pt item on the
+%    vertical list and if that isn't removed then the package doesn't
+%    find its penalties.
+%
+%    So this this needs to be removed as well.
```
Comment (u-fischer) [inline required/firstaid/latex2e-first-aid-for-external-files.dtx:576]: one "this"  too much

**Comment by car222222** [comment]
Hope I am looking at the latest version of
.../latex2e-first-aid-... .dtx ,
% \subsection{\cs{varwidth} first aid}

Two small things:

%    manipulation assuming traditional \TeX{} paragraphs. However with

==>  However, with 

%    So this needs to be removed as well by adding and additional

==>  an additional

**Comment by FrankMittelbach** [comment]
you did and I updated

**Pull request merged**